### ISSUE TITLE
About group normalization

### ISSUE THREAD

**Issue by RuoyiDu** [YingqingHe/ScaleCrafter#2]
Great work! Fixing the seam problem of tiled decoder by group normalization is a good idea, but why I do not find the relevant code in the repo?

**Comment by YingqingHe** [comment]
Hi, thanks for your comments! We will update this part of the code recently. please stay tuned~

**Comment by zwishenzug** [comment]
I'd also love to see the code for that, or have it explained in a bit more detail.

I'm trying to replace the function in the model:

    def Normalize(in_channels, num_groups=32):
        return torch.nn.GroupNorm(num_groups=num_groups, num_channels=in_channels, eps=1e-6, affine=True)

with a custom class instead, which has different modes, for a first pass where it collects statistics as each tile is passed through, then a second pass where it applies those statistics onto the tiles as they are run again.  But I cannot get it to work and just don't really understand what I should be doing from reading the paper.

**Comment by YingqingHe** [comment]
Hi~ We have updated this part of the code at the lightning version. Please check this link: https://github.com/YingqingHe/ScaleCrafter-ptl#-tiled-vae-decoding-with-syncgn. The diffuser version of this part is refactoring, please stay tuned. Hope this could help. Thanks!

**Comment by zwishenzug** [comment]
> Hi~ We have updated this part of the code at the lightning version. Please check this link: https://github.com/YingqingHe/ScaleCrafter-ptl#-tiled-vae-decoding-with-syncgn. The diffuser version of this part is refactoring, please stay tuned. Hope this could help. Thanks!

Thank you!  I will check it out now.

**Comment by ssyang1999** [comment]
Hi. Diffusers sync GN is ready. SD 1.5 and SD 2.1 scripts now activate this feature by default. Please see https://github.com/YingqingHe/ScaleCrafter/blob/main/sync_tiled_decode.py
